### PR TITLE
PR Materials March 2020

### PR DESCRIPTION
**Pull request by Loji3**
Transferring this over from Google Drive

PREVIEW: https://github.com/lindseymmc/pr/blob/patch-1/pr-materials-march-2020.md

### CODE REVIEW

**Comment by xaur** [comment]
I made some changes that I felt are non controversial and straight improvements. Please review them in this commit: https://github.com/decredcommunity/pr/pull/7/commits/6982485ea638761f46dd459ac5fee5996350497c

Note that we can revert any of them.

Below is rationale for some of my changes.

> It has already implemented the Lightning Network on mainnet

I changed to "ported and launched" to be more specific and harder to interpret as under-crediting enormous work on LN done by Lightning Labs.

> only non-btccore project to do so

I changed to "non-Bitcoin Core" because "btccore" is not a commonly used name.

> btcsuite code is the basis of LN, first atomic swap, one of the first LN implementations.

1. "first atomic swap" looks odd in between two parts of what looks like a single item about LN
2. To be a bit more specific, btcsuite helped to build `lnd`, which is an implementation of LN by Lightning Labs.

I updated text to reflect the above.

> Every Decred community member with "skin in the game" - stakeholders, developers, and miners - can vote on the direction of the project.

Only ticket holders can explicitly and directly vote. I reworded this.

**Comment by xaur** [comment]
I don't like the emphasis on "buy DCR". I cannot say I want people to just buy it. I want people to understand how money works and how Decred works, and figure what's best for them. I'd replace "buy DCR" with "use DCR". This includes both SoV, MoE and the rest, and also doesn't have the needy/desperate "please buy" vibe.

> We recently added a market maker on various trading pairs at the most reputable exchanges.

This is debatable, e.g. for me the most reputable exchange is Kraken. I suggest to drop "most".

> We expect a Coinbase listing soon, as CB [announced](https://blog.coinbase.com/coinbase-continues-to-explore-support-for-new-digital-assets-4d2ecbcbd38c) in August of 2019

1. This is pure speculation on CB "considering" things. I don't know why they post about "considering" at all.
2. This reads needy/desperate ("please list us!").
3. This is not cypherpunk/OG for the coin to dream of listings: cryptocurrencies are primary while fiat gateways are secondary (and hopefully they will become optional). It would be hard but cryptocurrencies can exist without exchanges while crypto exchanges cannot exist without cryptocurrencies. It takes orders of magnitude more commitment to crypto values and expertise to build a cryptocurrency than to build an exchange. I hope it's only a matter of time when exchanges realize their place. We can already see this as people become disillusioned about listings and crazy listing fees become thing of the past.
4. Most exchanges are "not OG": They will do anything to keep the business afloat. Coinbase makes a huge contribution to removing custody and privacy from cryptocurrency users. And do you believe they [bought Neutrino](https://breakermag.com/coinbases-newest-team-members-helped-authoritarians-worldwide-monitor-journalists-and-dissidents/) without knowing who's behind it?
5. Concentration of coins on exchanges is especially bad for DCR because coin holders control consensus.

> [Decred DEX](https://github.com/decred/dcrdex) alpha is expected to go live by April of 2020

May not happen. Like all software development, Decred is bad at projecting release dates. I would replace with just "2020".

> which will bring strong liquidity to DCR/BTC

Too speculative, due to hardcore UX it may not add significant liquidity, at least initially.

> Decred is best positioned as an alternative to Bitcoin for SoV, MoE and UoA. It is more secure

If by "secure" we mean attack cost, it is only more secure than Bitcoin given enough hashrate and market cap. Today Decred is less expensive to attack.

> only non-Bitcoin Core project to do so

This may not be correct and needs checking. IIRC at least Litecoin has LN and it may predate Decred's LN. In any case, I would reword this to remove the "only".

This statement is repeated mutliple times throughout the doc.

> It has SPV, which makes the blockchain size small enough to run a node on a mobile device, enabling mobile payments directly from the wallet.

1. Incorrect, SPV does not alter the blockchain size. It allows "light clients" that don't have to store the entire blockchain (~4 GB now) and only need to store a small subset of blockchain data (~400 MB). I would use "makes storage requirement small enough" or something like that.
2. Mobile devices don't "run a node" in a sense of a "full node", which need to store the full blockchain. They run "lightweight node" but I would remove "node" from here at all to avoid confusion.
3. "directly from the wallet" reads odd, I would change to "enabling payments from mobile wallets"
4. Decred not only has "just" SPV (which commonly refers to Electrum-like SPV in Bitcoin), but privacy-respecting decent SPV based on "compact filters". The difference with Electrum is huge: Decred SPV wallets connect to Decred full nodes directly without buggy/insecure/surveiling middleware servers.

> work for the blockchain

I don't want to work for the soulless blockchain, I want to work for like-minded people. Deemphasizing people in favor of fancy tech makes no sense to me.

> Decred has a self-sustaining treasury fund, you know you will get paid

This may come across as an overpromise. Contributors are required to demonstrate a fair amount of persistence and risk tolerance before they get their first payment. And then, it depends on location ofc, but in some cases developers are paid below market rates.

> Work with a team of technical rock stars

I think this is true but personally, "rock stars" has a vibe of this weird modern bs hipster culture where people call themselves "rock stars", "ninjas", "samurais", "gurus", etc while they are not really good enough (at least in my rankings).

Speaking of culture I'd like Decred to associate with is the [hacker culture](https://en.wikipedia.org/wiki/Hacker_culture), and I'd like Decred to stay humble.

> Build the future of money.

Yes! Let's have more of this!

> Bleeding edge of blockchain: Decred developers launched [btcsuite](https://github.com/btcsuite) ...

btcsuite is also a very readable (compared to C++ Bitcoin Core code) and well-documented codebase that can be used to educate how Bitcoin works.

> add custody

This is debatable but personally I am turning against the promotion of delegated custody services. Not your keys not your coins. For Decred this can go especially bad because coin holders control consensus.

> How we address them: Most of them subscribe to the traditional VC models, so they're looking for "use cases" and listening to what others are saying and doing. Most of them act as a flock, and it's a matter of reaching the tipping point when enough of them subscribe that they feel FOMO and do the same.

I don't think we should target this group. Decred has direct voting which requires intelligent coin holders who can think independently.

If someone is looking for "use cases" for money and doesn't see the "money" use case, they need to learn more before buying DCR.

> Superior Store of Value

Candidate.

> Borderless: Russia, China, ...

What does "borderless" mean here? Needs some clarification.

> Thanks to Anderson Cooper's "60 Minutes" segment, Bitcoin got its 15 minutes of fame in the mainstream media,

A link would be nice.

> Decred aims to deliver incremental privacy without risking stealth inflation (the blockchain must be auditable at all times), which is unlike other privacy coins that have taken more extreme measures and attracted dark markets

Not sure how "attracted dark markets" is connected to measures other privacy coins have taken. I would remove this or clarify.

This "dark markets" thing is also in "Unique Value Proposition" section.

> Code is written by contractors

Not necessarily, not all contributors are paid contractors. Perhaps "contributors" is better.

> - Rebellious

Not sure we need this one. Decred is building a better, alternative. opt-in system. It is not directly fighting existing system.

> - Decred is a top cryptocurrency by market cap (36)

This is not too bad but sounds a bit "wanna-be". When I hear "top" I imagine "top 5" or "top 10". Arguable ofc.

> - Decred is known as the project with the most bitcoin developers

I don't think this is true and I'm sure it will backfire. If keeping this, it must be backed by a link to strong evidence.

> - Decred is a fork-resistant store of value, proven to be twenty times more expensive to attack than Bitcoin due to its hybrid PoW/PoS model

Theorized, not proven. There was no real test yet.

> Completed the first stakeholder-voted consensus change vote in the history of cryptocurrency

It _may_ be true but also it may be not. For example, if pure PoS coins like Peercoin or masternode-based ones like Dash had coin-holder-driven hard forks before Decred's first vote in Apr 2017, Decred is not "first in the history". This claim must be checked and backed, and/or clarified, or removed.

> Decred is a completely decentralized organization

Not yet. This is repeated in "Unique Value Proposition".

> - Politeia average voter turnout is 32%, average participation for consensus rule changes is greater than 50%.  
> - Most of the other blockchain projects have voter participation around 1-3%, sometimes less.

Decred's numbers represents ~16% and ~25% of circulating coin supply because roughly half of the supply participates in PoS and has tickets to vote. I would note this because some projects talk about their turnout in terms of the entire circulating supply. Comparing apples to apples is a fair way to compare projects, or even the only way that makes sense.

> - Privacy: 100% of Monero is private (default). Last I checked 3% of Zcash is private.

Zcash numbers need to be updated, they are making efforts to improve the private %.

> - First-ever crypto atomic swap, between Decred (DCR) and Litecoin (LTC)

Atomic swaps before that DCR-LTC swap may have happened. Ideally it needs to be clarified what exactly was "first", i.e. publically available and maintained code for doing the swaps.

> - Built the first-ever off-chain proposal system (Politeia) that allows for public discourse and voting on project-level decisions and how to spend the treasury.

Dash's proposal site may have been first. IIRC it was just a regular forum-like site. I think where Decred's proposal system was "first" is by having open data (stored in public Git repo, not in a private MySQL database -> anybody can get a copy and detect games), integrity protection (anchored to blockchain, extremely hard to quietly alter) and a way to prove censorship. Perhaps @RichardRed0x can comment on the Dash site.

> First project to build a truly decentralized exchange (DEX)

1. It is not "truly" decentralized, trades will be coordinated through a central server
2. It is not built yet

> - Automate Treasury.

I think decentralizing the Treasury is more important than automating it.

> - Infographics: The design team regularly generates [images](https://github.com/decred/dcrdesign/releases) to be used for each software release or integration.

Our release images are not infographics, they do not visualize data or insights.

> - Evolving workspace. Work on what you want, when you want, work autonomously for the blockchain.

I would add "and get paid anywhere in the world without the banking hassles" and replace "work for the blockchain" with something more human (as noted earlier).

> decentralized exchange ruled by the collective intelligence of the community

This may read like the community will control DEX servers, which is not true. DEX servers will be controlled by their operators. The general direction and construction of DEX software is approved by the stakeholders though.

> **Sustainability**: Self-funding treasury - 10% of every block is awarded to the treasury. Decred is/was the first cryptocurrency to do this.

Decred is not the first to send a portion of block rewards to a treasury.

**Comment by xaur** [comment]
This document seems to copy a lot from foundational messaging. It also contains multiple repetitions of certain fragments. I would like to avoid the duplication of information but not yet sure how to approach it without losing the desired properties of the document, or keep it as is.

**Comment by xaur** [comment]
@lindseymmc sorry I forgot to mention one important thing: while I maintain this repository and try to keep the docs organized, I'm not a PR or Marketing lead and I will not block the merge of this document.

I shared my feedback because it may serve as a useful input for PR, but it is just one personal opinion among dozens of others you might hear in the community on this subject.

Let me know when you think the version in this branch looks good for merging, and any remaining items we can move to the next iteration.

**Comment by xaur** [comment]
Also, despite my long list of suggestions above, I didn't mention that the doc got a lot of things right and is already very helpful in its current form.

**Comment by lindseymmc** [comment]
> I made some changes that I felt are non controversial and straight improvements. Please review them in this commit: [4b5930a](https://github.com/decredcommunity/pr/commit/4b5930a2a93b1733a4f64277df0d9f3dfea53f5d)
> 
> Note that we can revert any of them.
> 
> Below is rationale for some of my changes.
> 
> > It has already implemented the Lightning Network on mainnet
> 
> I changed to "ported and launched" to be more specific and harder to interpret as under-crediting enormous work on LN done by Lightning Labs.
> 
> > only non-btccore project to do so
> 
> I changed to "non-Bitcoin Core" because "btccore" is not a commonly used name.
> 
> > btcsuite code is the basis of LN, first atomic swap, one of the first LN implementations.
> 
> 1. "first atomic swap" looks odd in between two parts of what looks like a single item about LN
> 2. To be a bit more specific, btcsuite helped to build `lnd`, which is an implementation of LN by Lightning Labs.
> 
> I updated text to reflect the above.
> 
> > Every Decred community member with "skin in the game" - stakeholders, developers, and miners - can vote on the direction of the project.
> 
> Only ticket holders can explicitly and directly vote. I reworded this.

**Comment by Loji3** [comment]
> I made some changes that I felt are non controversial and straight improvements. Please review them in this commit: [4b5930a](https://github.com/decredcommunity/pr/pull/7/commits/6982485ea638761f46dd459ac5fee5996350497c)
> 
> Note that we can revert any of them.
> 
> Below is rationale for some of my changes.
> 
> > It has already implemented the Lightning Network on mainnet
> 
> I changed to "ported and launched" to be more specific and harder to interpret as under-crediting enormous work on LN done by Lightning Labs.
> 
> > only non-btccore project to do so
> 
> I changed to "non-Bitcoin Core" because "btccore" is not a commonly used name.
> 
> > btcsuite code is the basis of LN, first atomic swap, one of the first LN implementations.
> 
> 1. "first atomic swap" looks odd in between two parts of what looks like a single item about LN
> 2. To be a bit more specific, btcsuite helped to build `lnd`, which is an implementation of LN by Lightning Labs.
> 
> I updated text to reflect the above.
> 
> > Every Decred community member with "skin in the game" - stakeholders, developers, and miners - can vote on the direction of the project.
> 
> Only ticket holders can explicitly and directly vote. I reworded this.

@xaur I reviewed all of the text changes and everything looks great, thank you for fact-checking and reviewing this so thoroughly! This is the link I was referring to for the stats: https://dcrdata.decred.org

How do we update the document with these changes, is this something I do or you?

**Comment by Loji3** [comment]
> I don't like the emphasis on "buy DCR". I cannot say I want people to just buy it. I want people to understand how money works and how Decred works, and figure what's best for them. I'd replace "buy DCR" with "use DCR". This includes both SoV, MoE and the rest, and also doesn't have the needy/desperate "please buy" vibe.

Agree that this would be a good change. Let's update it to 'Use DCR as a SoV, MoE etc.  

> > We recently added a market maker on various trading pairs at the most reputable exchanges.
> 
> This is debatable, e.g. for me the most reputable exchange is Kraken. I suggest to drop "most".

Will do 

> > We expect a Coinbase listing soon, as CB [announced](https://blog.coinbase.com/coinbase-continues-to-explore-support-for-new-digital-assets-4d2ecbcbd38c) in August of 2019
> 
> 1. This is pure speculation on CB "considering" things. I don't know why they post about "considering" at all.
> 2. This reads needy/desperate ("please list us!").
> 3. This is not cypherpunk/OG for the coin to dream of listings: cryptocurrencies are primary while fiat gateways are secondary (and hopefully they will become optional). It would be hard but cryptocurrencies can exist without exchanges while crypto exchanges cannot exist without cryptocurrencies. It takes orders of magnitude more commitment to crypto values and expertise to build a cryptocurrency than to build an exchange. I hope it's only a matter of time when exchanges realize their place. We can already see this as people become disillusioned about listings and crazy listing fees become thing of the past.
> 4. Most exchanges are "not OG": They will do anything to keep the business afloat. Coinbase makes a huge contribution to removing custody and privacy from cryptocurrency users. And do you believe they [bought Neutrino](https://breakermag.com/coinbases-newest-team-members-helped-authoritarians-worldwide-monitor-journalists-and-dissidents/) without knowing who's behind it?
> 5. Concentration of coins on exchanges is especially bad for DCR because coin holders control consensus.

Our reason for mentioning this is to address audience pain points/concerns about ease and convenience of acquiring DCR.

> > [Decred DEX](https://github.com/decred/dcrdex) alpha is expected to go live by April of 2020

Again this point is to address audience concerns of few fiat on-ramps, no Coinbase listing 

> May not happen. Like all software development, Decred is bad at projecting release dates. I would replace with just "2020".

It would be good to give a rough indication of timing - Spring/Summer 2020? 

> > which will bring strong liquidity to DCR/BTC
> 
> Too speculative, due to hardcore UX it may not add significant liquidity, at least initially.
> What about changing to 'should bring strong liquidity'
> > Decred is best positioned as an alternative to Bitcoin for SoV, MoE and UoA. It is more secure
> 
> If by "secure" we mean attack cost, it is only more secure than Bitcoin given enough hashrate and market cap. Today Decred is less expensive to attack.

Secure in terms of attack cost and also fork-resistant 

> > only non-Bitcoin Core project to do so
> 
> This may not be correct and needs checking. IIRC at least Litecoin has LN and it may predate Decred's LN. In any case, I would reword this to remove the "only".

Okay I'll remove 'only' and look into this

> This statement is repeated mutliple times throughout the doc.

I'll update all of them 

> > It has SPV, which makes the blockchain size small enough to run a node on a mobile device, enabling mobile payments directly from the wallet.
> 
> 1. Incorrect, SPV does not alter the blockchain size. It allows "light clients" that don't have to store the entire blockchain (~4 GB now) and only need to store a small subset of blockchain data (~400 MB). I would use "makes storage requirement small enough" or something like that.
> 2. Mobile devices don't "run a node" in a sense of a "full node", which need to store the full blockchain. They run "lightweight node" but I would remove "node" from here at all to avoid confusion.
> 3. "directly from the wallet" reads odd, I would change to "enabling payments from mobile wallets"
> 4. Decred not only has "just" SPV (which commonly refers to Electrum-like SPV in Bitcoin), but privacy-respecting decent SPV based on "compact filters". The difference with Electrum is huge: Decred SPV wallets connect to Decred full nodes directly without buggy/insecure/surveiling middleware servers.

Thank your this feedback. How about this: 'Decred has privacy-respecting decent SPV based on "compact filters, which makes storage requirement small enough to run on a mobile device, enabling payments from mobile wallets.'

> > work for the blockchain
> 
> I don't want to work for the soulless blockchain, I want to work for like-minded people. Deemphasizing people in favor of fancy tech makes no sense to me.

@Dustorf I think you added 'work for the blockchain' what's your perspective on this? 

> > Decred has a self-sustaining treasury fund, you know you will get paid
> 
> This may come across as an overpromise. Contributors are required to demonstrate a fair amount of persistence and risk tolerance before they get their first payment. And then, it depends on location ofc, but in some cases developers are paid below market rates.

@Dustorf could you please share some perspective on this? Was it to address other projects where it not guaranteed or work is unpaid? 
> 
> > Work with a team of technical rock stars
> 
> I think this is true but personally, "rock stars" has a vibe of this weird modern bs hipster culture where people call themselves "rock stars", "ninjas", "samurais", "gurus", etc while they are not really good enough (at least in my rankings).

I guess this is pretty subjective, what would be the best way to quantify this? A team of the most skilled, dedicated software engineers, marketers, creatives etc. in the blockchain space? 
> 
> Speaking of culture I'd like Decred to associate with is the [hacker culture](https://en.wikipedia.org/wiki/Hacker_culture), and I'd like Decred to stay humble.

Good to know. We can include this point about culture. 
> 
> > Build the future of money.
> 
> Yes! Let's have more of this!
> 
> > Bleeding edge of blockchain: Decred developers launched [btcsuite](https://github.com/btcsuite) ...
> 
> btcsuite is also a very readable (compared to C++ Bitcoin Core code) and well-documented codebase that can be used to educate how Bitcoin works.

Thanks I will add this

> > add custody

Where? 

> This is debatable but personally I am turning against the promotion of delegated custody services. Not your keys not your coins. For Decred this can go especially bad because coin holders control consensus.
> 
> > How we address them: Most of them subscribe to the traditional VC models, so they're looking for "use cases" and listening to what others are saying and doing. 
> 
> I don't think we should target this group. Decred has direct voting which requires intelligent coin holders who can think independently.
> 
> If someone is looking for "use cases" for money and doesn't see the "money" use case, they need to learn more before buying DCR.
> 
> > Superior Store of Value
> 
> Candidate.
> 
> > Borderless: Russia, China, ...
> 
> What does "borderless" mean here? Needs some clarification.

DCR being a borderless cryptocurrency would be attractive to these countries - @Dustorf is that correct?
> 
> > Thanks to Anderson Cooper's "60 Minutes" segment, Bitcoin got its 15 minutes of fame in the mainstream media,
> 
> A link would be nice.

@Dustorf is this the link you were referencing? https://www.cbsnews.com/news/bitcoins-wild-ride-60-minutes-2019-05-19/
> 
> > Decred aims to deliver incremental privacy without risking stealth inflation (the blockchain must be auditable at all times), which is unlike other privacy coins that have taken more extreme measures and attracted dark markets
> 
> Not sure how "attracted dark markets" is connected to measures other privacy coins have taken. I would remove this or clarify.

Okay I'm not sure which doc this was pulled from so I'll remove 
> 
> This "dark markets" thing is also in "Unique Value Proposition" section.

Thanks I'll update
 
> > Code is written by contractors
> 
> Not necessarily, not all contributors are paid contractors. Perhaps "contributors" is better.

Okay I'll update

> > * Rebellious
> 
> Not sure we need this one. Decred is building a better, alternative. opt-in system. It is not directly fighting existing system.

Okay, I can remove unless there are other arguments to keep it
 
> > * Decred is a top cryptocurrency by market cap (36)
> 
> This is not too bad but sounds a bit "wanna-be". When I hear "top" I imagine "top 5" or "top 10". Arguable ofc.

I see your point. My thinking is that given there are nearly 3,000 cryptocurrencies, it's adds instant credibility to highlight that we're in the top 35/50. 
 
> > * Decred is known as the project with the most bitcoin developers
> 
> I don't think this is true and I'm sure it will backfire. If keeping this, it must be backed by a link to strong evidence.

It's probably best to remove this then. 
 
> > * Decred is a fork-resistant store of value, proven to be twenty times more expensive to attack than Bitcoin due to its hybrid PoW/PoS model
> 
> Theorized, not proven. There was no real test yet.

I will change to "Decred is a fork-resistant store of value, which would be twenty times more expensive to attack than Bitcoin due to its hybrid PoW/PoS model"

> > Completed the first stakeholder-voted consensus change vote in the history of cryptocurrency
> 
> It _may_ be true but also it may be not. For example, if pure PoS coins like Peercoin or masternode-based ones like Dash had coin-holder-driven hard forks before Decred's first vote in Apr 2017, Decred is not "first in the history". This claim must be checked and backed, and/or clarified, or removed.

I will look into this 

> > Decred is a completely decentralized organization
> 
> Not yet. This is repeated in "Unique Value Proposition".

What about "Decred is a decentralized organization"

> > * Politeia average voter turnout is 32%, average participation for consensus rule changes is greater than 50%.

> > * Most of the other blockchain projects have voter participation around 1-3%, sometimes less.

Thank you I will add 
 
> Decred's numbers represents ~16% and ~25% of circulating coin supply because roughly half of the supply participates in PoS and has tickets to vote. I would note this because some projects talk about their turnout in terms of the entire circulating supply. Comparing apples to apples is a fair way to compare projects, or even the only way that makes sense.

Thank you I will add

> > * Privacy: 100% of Monero is private (default). Last I checked 3% of Zcash is private.
> 
> Zcash numbers need to be updated, they are making efforts to improve the private %.

I will update

> > * First-ever crypto atomic swap, between Decred (DCR) and Litecoin (LTC)
> 
> Atomic swaps before that DCR-LTC swap may have happened. Ideally it needs to be clarified what exactly was "first", i.e. publically available and maintained code for doing the swaps.

I will check this

> > * Built the first-ever off-chain proposal system (Politeia) that allows for public discourse and voting on project-level decisions and how to spend the treasury.
> 
> Dash's proposal site may have been first. IIRC it was just a regular forum-like site. I think where Decred's proposal system was "first" is by having open data (stored in public Git repo, not in a private MySQL database -> anybody can get a copy and detect games), integrity protection (anchored to blockchain, extremely hard to quietly alter) and a way to prove censorship. Perhaps @RichardRed0x can comment on the Dash site.

I'll get clarification on this

> > First project to build a truly decentralized exchange (DEX)
> 
> 1. It is not "truly" decentralized, trades will be coordinated through a central server
> 2. It is not built yet

Is the end goal to build a truly decentralized exchange? 
 
> > * Automate Treasury.
> 
> I think decentralizing the Treasury is more important than automating it.

I'll update to "first to decentralize treasury" 

> > * Infographics: The design team regularly generates [images](https://github.com/decred/dcrdesign/releases) to be used for each software release or integration.
> 
> Our release images are not infographics, they do not visualize data or insights.

Got it

> > * Evolving workspace. Work on what you want, when you want, work autonomously for the blockchain.
> 
> I would add "and get paid anywhere in the world without the banking hassles" and replace "work for the blockchain" with something more human (as noted earlier).

Will do 

> > decentralized exchange ruled by the collective intelligence of the community
> 
> This may read like the community will control DEX servers, which is not true. DEX servers will be controlled by their operators. The general direction and construction of DEX software is approved by the stakeholders though.

Okay I can take this out

> > **Sustainability**: Self-funding treasury - 10% of every block is awarded to the treasury. Decred is/was the first cryptocurrency to do this.
> 
> Decred is not the first to send a portion of block rewards to a treasury.

I'll update. Thank you again for taking the time to review this - much appreciated!

**Comment by Loji3** [comment]
> This document seems to copy a lot from foundational messaging. It also contains multiple repetitions of certain fragments. I would like to avoid the duplication of information but not yet sure how to approach it without losing the desired properties of the document, or keep it as is.

The purpose of the messaging section is as follows: 

- To gather all the key information that we want to communicate externally in one place 
- To check it's all correct
- To agree we're happy this is what we want to communicate (is the messaging correct? Are there any great stats or points we're missing?)

I will be using this document to draw up press releases, media pitches, reactive statements, talking points etc. so It's important to have things like key stats, credibility points etc. together in an easy-to-reference place, even though they've already been mentioned throughout the doc. It means I don't have to comb through the entire document each time picking them out. It's not a public-facing brochure or a document I'm going to be sharing in entirety to anyone externally.

**Comment by Loji3** [comment]
> @lindseymmc sorry I forgot to mention one important thing: while I maintain this repository and try to keep the docs organized, I'm not a PR or Marketing lead and I will not block the merge of this document.
> 
> I shared my feedback because it may serve as a useful input for PR, but it is just one personal opinion among dozens of others you might hear in the community on this subject.
> 
> Let me know when you think the version in this branch looks good for merging, and any remaining items we can move to the next iteration.

Sounds great. It's important we're sharing factual information so I appreciate the feedback. I'll share in the marketing channel to get some additional feedback, update the agreed upon changes then check back in with you when it's ready to merge. Thanks again!

**Comment by Loji3** [comment]
> Also, despite my long list of suggestions above, I didn't mention that the doc got a lot of things right and is already very helpful in its current form.

Good to hear thank you!

**Comment by xaur** [comment]
Pulled some formatting tweaks from the foundational messaging. This also reduces the diff with it.

**Comment by xaur** [comment]
@lindseymmc I made some changes on items I think we've agreed upon that you can review in https://github.com/decredcommunity/pr/pull/7/commits/51c840396182e1a7b93504454cf0abf1138534a3 (use Rich Diff to easier see them). Again, anything can be reverted or tweaked further if needed.

Change summary, in the order they occur in text:

- removed "dark markets" stuff
- relaxed "contractors" to "contributors" in a few places
- removed "Rebellious"
- removed "known as the project with the most Bitcoin developers"
- changed "proven to be twenty times more expensive" to "would be ~20x more expensive"
- changed "the only non-Bitcoin Core based project to implement Lightning Network" to "Decred is one of the few projects with the capacity to deploy Lightning Network"
- changed the "market cap" key stat to a safer "Market cap rank between 35-50 with cap value $100-200 million throughout 2019-2020"
- changed "Decred is a completely decentralized organization" to "Decred's goal is to become a completely decentralized organization"
- changed "First project to build a truly decentralized exchange" to "First project to fund the development of a truly decentralized exchange"
- changed "Automate Treasury" to "Decentralize Treasury control" and clarified that full decentralization is the goal but will be done carefully
- removed "ruled by the collective intelligence of the community" in regards to the DEX operation
- removed "Decred is/was the first cryptocurrency to do this" in relation to self-funding treasory.

> > Speaking of culture I'd like Decred to associate with is the hacker culture, and I'd like Decred to stay humble.
> 
> Good to know. We can include this point about culture.

This is my personal vision and I don't see any community members talking about it... yet (btw one thing to know about hackers is you don't call yourself a hacker).

> Is the end goal to build a truly decentralized exchange?

Yes, the end goal/aspiration is a truly decentralized exchange. But we're pragmatic and realize it may take years and years to achieve. Direct trustless trades is a huge step nevertheless.
